net.filebot.Filebot.appdata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>net.filebot.Filebot</id>
+  <name>Filebot</name>
+  <project_license>LicenseRef-proprietary</project_license>
+  <developer_name>rednoah</developer_name>
+  <summary>The ultimate TV and Movie Renamer</summary>
+  <url type="homepage">https://filebot.net</url>
+  <description>
+    <p>FileBot is the ultimate tool for renaming and organizing your movies, TV shows and Anime. Match and rename media files against online databases, download artwork and cover images, fetch subtitles, write metadata, and more, all at once in matter of seconds. It's smart and just works.</p>
+  </description>
+  <metadata_license>CC0-1.0</metadata_license>
+  <categories>
+    <category>AudioVideo</category>
+    <category>Video</category>
+    <category>Utility</category>
+    <category>FileTools</category>
+  </categories>
+  <releases>
+    <release version="4.9.5" date="2022-02-22"/>
+    <release version="4.9.4" date="2021-07-28"/>
+  </releases>
```

Comment:
missing `launchable`, `screenshots` and `content_rating`.

Reply:
Added launchable.
Added screenshots.
Added content_rating, guessed it needs to be empty. This can download strings of text, which potentially are affected by content rating, I'm not sure what to set here. If you can find a show, that has an episode name, that is offensive in any way, it will set that as the file name.